lib/opendata-tw.js

```diff
@@ -23,6 +23,7 @@ class OpendataTW {
 };
 
 // TODO: fetch method rather than url
-OpendataTW.url = 'https://data.ntpc.gov.tw/api/datasets/308DCD75-6434-45BC-A95F-584DA4FED251/json?size=2000';
+OpendataTW.urlFirst1000 = 'https://data.taipei/api/v1/dataset/b0011e96-3fc3-43ec-8bf5-07fb46dd22bb?scope=resourceAquire&q=&limit=1000=&offset=0';
+OpendataTW.urlSecond1000 = "https://data.taipei/api/v1/dataset/b0011e96-3fc3-43ec-8bf5-07fb46dd22bb?scope=resourceAquire&q=&limit=1000=&offset=1000";
```

Comment:
```suggestion
OpendataTW.urlSecond1000 = 'https://data.taipei/api/v1/dataset/b0011e96-3fc3-43ec-8bf5-07fb46dd22bb?scope=resourceAquire&q=&limit=1000=&offset=1000';
```

for quote consistency

Reply:
Great suggestion. I should notice the code style consistency and readability. I'll make another patch soon.

Reply:
I've add a patch to implement the suggestion. Should we continue the discussion of fetching method in issue #9 ?